tests/typegenPrinter.spec.ts

```diff
@@ -19,7 +20,12 @@ describe('typegenPrinter', () => {
           __typename: true,
         },
       },
-      prettierConfig: path.join(__dirname, '../.prettierrc'),
+      async formatTypegen(source, type) {
```

Comment:
This is a fix after merging the abstract type PR. Basically, since we added the `Discriminate<>` type helper, we `import { core } from @nexus/schema` when types are needed to be discriminated, which is the case on that test. So before, we weren't importing anything from @nexus/schema but now we do.

I know it's not related to that PR but it was creating some type errors in some places